lib/ftu/parsing.dart

```diff
@@ -0,0 +1,59 @@
+class Localization {
+  final List <Specialty> specialty;
+  final List <Industry > industry;
+
+  Localization({this.specialty, this.industry});
+
+  factory Localization.fromJson(Map<String, dynamic> json) {
+    return Localization(
+ 
+       
+      industry: parseIndustry(json),
+      specialty: parseSpecialty(json),
+      
+    );
+  }
+
+  
+  static List<Industry > parseIndustry(industryJson) {
```

Comment:
I see there's a couple of functions converting the `filters.json` to a list. Instead of doing this, make the file a `.dart` file and make it a `List` object directly.
```dart
List filters = const [{ ... }];
```